src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -1349,6 +1355,308 @@ describe('toolbar mode only features', () => {
         expect(globalDrawersWrapper.findDrawerById('global3')).toBeFalsy();
       });
     });
+
+    describe('expanded mode for global drawers', () => {
+      test('should set a drawer to expanded mode by clicking on "expanded mode" button', async () => {
+        const drawerId = 'global-drawer';
+        awsuiPlugins.appLayout.registerDrawer({
+          ...drawerDefaults,
+          ariaLabels: {
+            expandedModeButton: 'Expanded mode button',
+          },
+          id: drawerId,
+          type: 'global',
+          isExpandable: true,
+        });
+        const { wrapper, globalDrawersWrapper } = await renderComponent(<AppLayout />);
+
+        await delay();
```

Comment:
Why is there an extra delay? Was the delay inside `await renderComponent()` not enough?